### PR TITLE
Weekly PR

### PR DESCRIPTION
**Pull request by vedi**

### CODE REVIEW

File: SoomlaAndroidStore/src/com/soomla/store/data/StoreInfo.java [hunk 1]
```diff
@@ -72,21 +75,47 @@
      * metadata when the application loads. Bumping the version is done by returning a higher number
      * in {@link IStoreAssets#getVersion()}.
      */
-    public static void setStoreAssets(IStoreAssets storeAssets) {
```
Comment (borisspektor) [inline SoomlaAndroidStore/src/com/soomla/store/data/StoreInfo.java]: @EugeneButusov please put the private methods below setStoreAssets and have setStoreAssets below its comment

**Comment by EugeneButusov** [comment]
@borisspektor, done.

File: SoomlaAndroidStore/src/com/soomla/store/data/StoreInfo.java (hunk 1 above)
Comment (EugeneButusov) [inline SoomlaAndroidStore/src/com/soomla/store/data/StoreInfo.java]: @borisspektor, done.

**Comment by vedi** [comment]
@refaelos, @borisspektor, it's ready to merge

**Pull request merged**